### DISCUSSION TITLE
How to manage tags?

### DISCUSSION THREAD

**Discussion by Saintyang** [apache/gravitino#5257] [Ideas]
We have tag to manage meta objects , but how to manage tags? I have 2 ideas:
1.  introduce tag group to gravitino
2. allow create tag on tag

Any ideas?

**Comment by jerryshao** [comment]
Hi @Saintyang we don't support these two features currently, you can take a shot if you'd like to.

**Reply by Saintyang** [reply]
So in your opinion, which is a better design? 1 or 2

**Reply by jerryshao** [reply]
There's no good or bad for these two features, they're mainly based on the requirements and usage scenarios.

**Comment by mchades** [comment]
We already have some [basic operations on tags](https://gravitino.apache.org/docs/0.6.1-incubating/manage-tags-in-gravitino#tag-operations),  Could you please provide specific examples to illustrate the type of tag management you require?
